cob_twist_controller/src/cob_twist_controller.cpp

```diff
@@ -263,31 +288,64 @@ void CobTwistController::initInvDiffKinSolverParams()
         return;
     }
 
-    InvDiffKinSolverParams params;
-    params.damping_method = MANIPULABILITY;
-    params.constraint = WLN_JLA;
-    params.eps_truncation = 0.001;
-    params.damping_factor = 0.2;
-    params.lambda_max = 0.1;
-    params.w_threshold = 0.005;
-    params.base_compensation = false;
-    params.base_active = false;
-    params.base_ratio = 0.0;
-    params.limits_min = twist_controller_params_.limits_min;
-    params.limits_max = twist_controller_params_.limits_max;
-    params.limits_vel = twist_controller_params_.limits_vel;
-    params.k_H = 1.0;
-
-    params.frame_names.clear();
+    //TwistControllerParams params;
+    
+//    params.dof = twist_controller_params_.dof;
+//    params.chain_base_link = twist_controller_params_.chain_base_link;
+//    params.chain_tip_link = twist_controller_params_.chain_tip_link;
```

Comment:
remove if not needed here anymore...(of course we keep them in the TwistControllerParams struct - I assume, this was obsolete redundant initialization...)